tests/test_data.py

```diff
@@ -138,16 +138,17 @@ async def test_parse_new_report_event():
 
     cfg = TelliotConfig()
     cfg.main.chain_id = 5
-    tx_hash = "0xf7fb66b0c3961692cd9658ce4a8c5e73ba8fbc954676d417e815456337604797"
+    tx_hash = "0xb3e4fb09e3dcc7abc0f30181c5dc9ba7253d694de17d88c37c2bd4069f0eebdc"
 
     for endpoint in cfg.endpoints.find(chain_id=5):
         cfg.endpoints.endpoints.remove(endpoint)
 
-    endpoint = RPCEndpoint(5, "Goerli", "Infura", os.getenv("NODE_URL"), "etherscan.io")
+    endpoint = RPCEndpoint(
+        5, "Goerli", "Infura", "https://goerli.infura.io/v3/db7ce830b1224efe93ae3240f7aaa764", "etherscan.io"
```

Comment:
Thy API key?

Reply:
Yes, test wasn't loading api key though so I left it. We could debug it together